@ecomplus/storefront-components/src/html/ProductCard.html

```diff
@@ -128,4 +128,8 @@
       {{ error }}
     </div>
   </template>
+
+  <template>
+    <div ref="quickview"></div>
+  </template>
```

Comment:
```suggestion
  <div ref="quickview"></div>
```

Não entendi pq o `<template>`...